sections/usptx/uxptx.go

```diff
@@ -0,0 +1,166 @@
+package usptx
```

Comment:
I believe there's a typo on this file's name. Did you mean `usptx.go` instead of `uxptx.go`?